demos/background_subtraction_demo/cpp_gapi/README.md

```diff
@@ -0,0 +1,105 @@
+# G-API Background Subtraction Demo
+
+This demo shows how to perform background subtraction using G-API.
+
+> **NOTE**: Only batch size of 1 is supported.
+
+## How It Works
+The demo application expects an background matting model in the Intermediate Representation (IR) format.
```

Comment:
In fact, demo works with BG matting model which isn't available in OMZ

Reply:
Changed to `instance-segmentation-securicy-????`

Reply:
I suggest to add to the readme a full description of models which should be supported with inputs/outputs specifications, because even instance segmentation models in OMZ have different inputs/outputs. Also [this](https://github.com/TolyaTalamanov/open_model_zoo/blob/at/gapi_bgr_subtraction/demos/background_subtraction_demo/cpp_gapi/src/custom_kernels.cpp#L212) line processes alpha, not a binary mask that does not match instance segmentation based models.

Reply:
Postprocessing for instance-segmentation has been added (blend by using binary mask).
Supported only models `instance-segmentation-person-????` with any type of inputs/outputs.

Reply:
I doubt that any types of inputs/outputs are supported here. Please take a look into [this](https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/background_subtraction_demo/python/README.md?plain=1#L12) example of description. Even `Mask RCNN` based models may have different inputs/outputs depending on a framework and there are a lot of other architectures.

Reply:
Done